docs/sandbox/apps/unifi.md

```diff
@@ -0,0 +1,63 @@
+# Unifi Controller
+
+## What is it?
+
+[Unifi Controller](https://www.ui.com/download/unifi/){: target=_blank rel="noopener noreferrer" } software is a powerful, enterprise wireless software engine ideal for high-density client deployments requiring low latency and high uptime performance.
+
+
+| Details     |             |             |             |
+|-------------|-------------|-------------|-------------|
+| [:material-home: Project home ](https://www.ui.com/download/unifi/){: .header-icons target=_blank rel="noopener noreferrer" } | [:octicons-link-16: Docs](https://github.com/linuxserver/docker-unifi-controller/blob/master/README.md){: .header-icons target=_blank rel="noopener noreferrer" } | [:octicons-mark-github-16: Github:](https://github.com/linuxserver/docker-unifi-controller){: .header-icons target=_blank rel="noopener noreferrer" } | [:material-docker: Docker ](https://hub.docker.com/r/linuxserver/unifi-controller){: .header-icons target=_blank rel="noopener noreferrer" }|
+
+### 1. Installation
+
+``` shell
+
+sb install sandbox-unifi
+
+```
+
+### 2. URL
+
+- To access Unifi Controller, visit `https://unifi._yourdomain.com_`
+
+### 3. Setup
+
+  1. Visit the vaultwarden site at `https://unifi._yourdomain.com_`
```

Comment:
wrong name

Reply:
corrected in commit [801632f](https://github.com/saltyorg/docs/pull/35/commits/801632f25414e1f0e79af6a679f6ae1abafb07cd)